iambic/plugins/v0_1_0/azure_ad/models.py

```diff
@@ -140,6 +140,8 @@ def dict(
             exclude = required_exclude
         elif isinstance(exclude, set):
             exclude.update(required_exclude)
+        # elif isinstance(exclude, dict):
```

Comment:
Safe to assume this isn't needed anymore / wasn't removed temporarily for debugging purposes?

Reply:
Oops, yeah just removed it